src/content/docs/specs/episodes/index.mdx

```diff
@@ -0,0 +1,131 @@
+
+### Download status
+
+The `download_status` is a _declaration of intent_, not an indication of the current status. If a user downloads an episode on client A, this client passes on this value to the server and thereby to other clients. Client B can then:
+* download immediately
+* download later (e.g. as soon as a WiFi connection is availalbe)
+
+It is up to the implementers whether this applies both to automatic and manual downloads, or only to manual downloads.
+
+While an optional field, if `download_status` is supported both by the server and the client, the client is expected to respect this field value. If the client may not download due to space limitations or won't download at all, then it should not declare support for this field.
+
+:::note[Discussion details]
+See meeting notes from [2024-02-27](https://pad.funkwhale.audio/s/88C5eXrRq)
+:::
+
+## Timestamping changes to resolve field conflicts
+
+To prevent unresolvable conflicts at field level, apps are expected to record timestamps. These MUST be the _time at which a field value was changed in the client_ (by the user or system). More recent values take presedence. The time of sending by the client or processing by the client should _not_ be sent.
+
+:::note[Example]
+Scenario:
+1. User plays first quarter of an episode on client A, which is offline.
+2. User skips first quarter and starts playback until halfway through the episode on client B, which is offline.
+3. Client B comes online and synchronises changes.
+4. Client A comes online and synchronises changes.
+
+In this case, the changes made on client B should take presedence, as they were made later in time than the changes on client A, even though client A most recently synchronised its data.
+:::
+
+To enable this, all _data fields_ are nested objects with a `value` and a `timestamp` field:
+
+<Tabs syncKey="accepts">
+  <TabItem label="JSON">
+
+    ```json {8, 11} collapse={2-4}
+      {
+        "playback_position": {
+          "value": 15,
+          "timestamp": "2024-06-19T15:46"
+        }
+      }
+    ```
+
+  </TabItem>
+  <TabItem label="XML">
+
+    ```xml {8, 11} collapse={3-5}
+    <?xml version="1.0" encoding="UTF-8"?>
+      <playback_position>
+        <value>15</value>
+        <timestamp>2024-06-19T15:46</timestamp>
+      </playback_position>
+    ```
+
+  </TabItem>
+</Tabs>
+
+Timestamps are recorded in [ISO 8601 format]. When a new episode is created, timestamps are set to current time.
+
+:::note[Discussion details]
+See meeting notes from [2024-02-27](https://pad.funkwhale.audio/s/6mWuDexgz#Data-timestamps)
+
+**Important open discussion point:**
+
+Sending back & forth timestamps is not needed if client always first pulls before push, and assuming that the client stores the timestamps of these changes locally. Maybe we should note this as a requirement, rather than submitting the timestamps.
```

Comment:
It's hard to wrap my head around this, but I have the impression that we still need to send the timestamps around. Because the last sync date might be in the past, and you have a more recent change, but the server might have received more recent data from another client in the meantime. @mogwa1?